### ISSUE TITLE
LPIPS  evaluation should add `normalize=True`

### ISSUE THREAD

**Issue by zehongs** [Wangt-CN/DisCo#41]
Hi, thanks for the great work.
I noticed that the LPIPS evaluation does not include `normalize=True` given that the inputs are in the [0,1] range. Adding this would change the results from 0.292 to 0.339. Despite this increase, the result still remains significantly better than the baseline.

https://github.com/Wangt-CN/DisCo/blob/8cb938712c8f5cdf33190731f804c2c4a7a3f56a/tool/metrics/ssim_l1_lpips_psnr.py#L74-L97

LPIPS:
https://github.com/richzhang/PerceptualSimilarity/blob/31bc1271ae6f13b7e281b9959ac24a5e8f2ed522/lpips/lpips.py#L112-L115

**Comment by SHTU-YuxuanGe** [comment]
That's right, the result should be higher if using the right value range.

The result should be updated.
